### ISSUE TITLE
The learn-rate of transformer-base and big model

### ISSUE THREAD

**Issue by lkfo415579** [marian-nmt/marian-dev#578]
I was wondering why you guys set the learn-rate to **0.0003/0.0002** respectively (it is very low), the paper is **0.0007/0.0005**. Is it because of the batch generation stragegy (**mini-batch-fit**) different from others?

Or is it because of the WMT14 dataset, the best tuning setup will be warmup 16000 and 0.0003?

And can you guys give a hyper-parameters tuning tutorial? it would be very helpful for me tuning the hyper-params.

**Comment by emjotde** [comment]
By "the paper" you mean the original transformer paper? Take into account that they are doing some really hard to follow magic by multiplying the learning rate by `1/sqrt(model-dim)` which we don't do. So it's kinda hard to figure out their actual effective learning rate. 

With larger batches I started using learning rates around 0.0005 for transformer-big models, but I have not been going larger than that.

**Comment by emjotde** [comment]
As for the hyper-parameters this isn't actually that much to tune, we are still quite close to the Transformer paper otherwise. I am working on moving the training methods from my WMT19 research branch in a clean way into master, that will allow to make hyper-parameter choices based on seen training labels and effective batch size rather than memory-bound batch size. This makes a lot of things a lot easier.

**Comment by emjotde** [comment]
Closing this. Feel free to re-open if you have more questions here.
